pipelines/shopgate.catalog.getFilters.v1.json

```diff
@@ -0,0 +1,166 @@
+      }
+    ],
+    "output": [
+      {
+        "key": "filters",
+        "id": "1000"
+        }
+      ],
+    "steps": [
+      {
+        "type": "conditional",
+        "input": [
+          {
+            "key": "categoryId",
+            "id": "1",
+            "optional": true
+          }
+        ],
+        "expression": {
+          "ok": [
+            {
+              "name": "categoryId"
+            }
+          ]
+        },
+        "then": {
+          "id": "@shopgate/products",
+          "path": "@shopgate/products/products/getFilters.js",
+          "type": "extension",
+          "input": [
+            {
+              "id": "1",
+              "key": "categoryId",
+              "optional": true
+            },
+            {
+              "id": "2",
+              "key": "searchPhrase",
+              "optional": true
+            },
+            {
+              "id": "3",
+              "key": "filters",
+              "optional": true
+            }
+          ],
+          "output": [
+            {
+              "id": "10",
+              "key": "service"
+            },
+            {
+              "id": "11",
+              "key": "version"
+            },
+            {
+              "id": "12",
+              "key": "path"
+            },
+            {
+              "id": "13",
+              "key": "method"
+            },
+            {
+              "id": "14",
+              "key": "query"
+            }
+          ]
+        }
+      },
+      {
+        "type": "conditional",
+        "input": [
+          {
+            "key": "categoryId",
+            "id": "1",
+            "optional": true
+          }
+        ],
+        "expression": {
+          "ok": [
+            {
+              "name": "categoryId"
+            }
+          ]
+        },
+        "then": {
+          "id": "@shopgate/bigapi",
+          "path": "@shopgate/bigapi/big-api/getBigApiResult.js",
+          "type": "extension",
+          "input": [
+            {
+              "id": "10",
+              "key": "service"
+            },
+            {
+              "id": "11",
+              "key": "version"
+            },
+            {
+              "id": "12",
+              "key": "path"
+            },
+            {
+              "id": "13",
+              "key": "method"
+            },
+            {
+              "id": "14",
+              "key": "query"
+            }
+          ],
+          "output": [
+            {
+              "id": "1000",
+              "key": "filters"
+            }
+          ]
+        },
+        "else": {
+          "type": "extension",
+          "id": "@shopgate-project/doofinder-search",
+          "path": "@shopgate-project/doofinder-search/lib/getFilters.js",
+          "input": [
+            {
+              "key": "searchPhrase",
+              "id": "2",
+              "optional": true
+            },
+            {
+              "key": "filters",
```

Comment:
The step does not accept `input.filters` we do not need to pass it

Reply:
not yet, but we talked about the nice to have to call the pipeline again when filter selection was made to refine the filters...
however this can be added in later as well, so I'll remove for now.